File: Apex Legends SDK/x64/Release/UserScripts/Glow.py
import Entity

#Apex Legends Health-based Glow ESP

def main():
    #pLocal = Entity.LocalEntity() #We dont need it for this script, but this is how you would get the localPlayer

    for i in range(0, 100):
        pEntity = Entity.BaseEntity(i)
        if pEntity.isPlayer() and pEntity.iHealth() > 0: #Valid Player
            #pEntity.GlowStyle(101,101,46,90)

            health = pEntity.iHealth() + pEntity.iShield()
            bleedoutstate = pEntity.isKnocked()

            if 180 <= health <= 300: #WHITE
                pEntity.Glow(255, 255, 255)
            if 160 <= health <= 179: #GREEN
                pEntity.Glow(0, 255, 0)
            if 140 <= health <= 159: #YELLOW
                pEntity.Glow(255, 255, 0)
            if 110 <= health <= 139: #RED
                pEntity.Glow(255, 0 , 0)
            if 70 <= health <= 109: #PINK
                pEntity.Glow(255, 0, 255)
            if 40 <= health <= 69: #BLUE
                pEntity.Glow(0, 0, 255)
            if health < 40:        #CYAN
                pEntity.Glow(0, 255, 255)
            if bleedoutstate != 0: #purple
                pEntity.Glow(136, 0, 255)

File: Apex Legends SDK/Apex Legends SDK/Main/Modules/Glow.cpp
#include "Modules.h"

void Modules::Glow(BaseEntity pEntity, LocalEntity pLocal)
{
	//pEntity.glowStyle = { 101,101,46,90 }; //Bright Glow, default is just outline

	int health = pEntity.iHealth();
	int bleedoutstate = pEntity.isKnocked();

	if (pEntity.iTeam() == pLocal.iTeam()) // DON'T GLOW ALLIES
	return;

	if(health <= 300 && health > 180)
	{
		pEntity.Glow(255.f, 255.f, 255.f); //white
	}
	else if (health <= 179 && health >  160)
	{
		pEntity.Glow(0.f, 255.f, 0.f); //green
	}
	else if (health <= 159 && health > 140)
	{
		pEntity.Glow(255.f, 255.f, 0.f); //yellow
	}
	else if (health <= 139 && health > 110)
	{
		pEntity.Glow(255.f, 0.f, 0.f); //red
	}
	else if (health <= 109 && health > 70)
	{
		pEntity.Glow(255.f, 0.f, 255.f); //pink
	}
	else if(health <= 69 && health > 40)
	{
		pEntity.Glow(0.f, 0.f, 255.f); //blue
	}
	else if (health < 40)
	{
		pEntity.Glow(0.f, 255.f, 255.f); //cyan
	}
	else if (bleedoutstate != 0)
	{
		pEntity.Glow(136.f, 0.f, 255.f); //purple
	}
}